### PR TITLE
101 aci vnet

### PR DESCRIPTION
**Pull request by tilnl**
ACI arm template with VNet

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-aci-vnet/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            }
+          }
+        ],
+        "osType": "Linux",
+        "networkProfile": {
+          "Id": "[resourceId('Microsoft.Network/networkProfiles', parameters('networkProfileName'))]"
+        },
+        "restartPolicy": "Always"
+      },
+      "dependsOn": [
```
[the remark this answers is not in the document]
Comment (tilnl) [reply 101-aci-vnet/azuredeploy.json]: @bmoore-msft thanks for the comments. Resolved all of them. Pls help to review again.

File: 101-aci-vnet/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                      "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('vnetName'), parameters('subnetName'))]"
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      },
+      "dependsOn": [
```
[the remark this answers is not in the document]
Comment (bmoore-msft) [reply 101-aci-vnet/azuredeploy.json]: this still needs to be moved

File: 101-aci-vnet/azuredeploy.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmoore-msft) [reply 101-aci-vnet/azuredeploy.json]: this still needs to be moved above the properties object

**Comment by tilnl** [comment]
@bmoore-msft The "dependsOn" properties were already fixed last week. Looks like you are commented on an outdated file. Could you please help to check again?

**Comment by tilnl** [comment]
@bmoore-msft Thanks!

**Pull request merged**